bitcoin/src/pow.rs

```diff
@@ -1077,7 +1077,9 @@ pub mod test_utils {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::pow::test_utils::{u128_to_work, u32_to_target, u64_to_target};
+    #[cfg(feature = "std")]
+    use crate::pow::test_utils::u128_to_work;
+    use crate::pow::test_utils::{u32_to_target, u64_to_target};
```

Comment:
Better to just remove the 'std' feature gate from the code that uses `u128_to_work`. (I tested it.)

Reply:
Oo, nice! Maybe let's wait for #4755 to land though before removing std gates from tests.